pkg/resmgr/nri.go

```diff
@@ -492,35 +513,137 @@ func (p *nriPlugin) updateResources(u *api.ContainerUpdate, c cache.Container) {
 	}
 }
 
-func (p *nriPlugin) dump(header string, args ...interface{}) {
-	for {
-		var (
-			prefix string
-			msg    []byte
-			ok     bool
-			err    error
-		)
+const (
+	request  = "=>"
+	response = "<="
+)
+
+func (p *nriPlugin) dump(dir, event string, args ...interface{}) {
+	// request dumping
+	if dir == request {
+		switch {
```

Comment:
One thing I forgot to ask is, why do we call the first argument `dir`? I was a bit confused initially, because I had an assumption that it is some kind of a path but then I realized that it is string which in our case can be either request or response.

Reply:
It is a 'direction', a request or a response. Perhaps 'kind' would be better, since we can't use 'type'. I'll update...

Reply:
I replaced request/response with in/out, adding const for them. Kept dir as such, since it should now be more obvious what it means.